docs/detections/rules-ui-monitor.asciidoc

```diff
@@ -69,6 +69,14 @@ You can also use Task Manager in {kib} to troubleshoot background tasks and proc
 * {kibana-ref}/task-manager-health-monitoring.html[Task Manager health monitoring]
 * {kibana-ref}/task-manager-troubleshooting.html[Task Manager troubleshooting]
 
+[float]
+[[troubleshoot-max-alerts]]
+==== Troubleshoot max alerts warning
+
+When a rule reaches the maximum number of alerts it can generate in a single rule execution, the following warning is displayed on the rule's details page and in the rule execution log: `This rule reached the maximum alert limit for the rule execution. Some alerts were not created.` 
+
+If you receive this warning, check the alerts and determine whether they should've been created. If any were wrongfully created, start investigating what caused the excess alerts. For example, check the rule's data source for possible issues or modify the rule's query if it's too broad in scope.
```

Comment:
> If you receive this warning, check the alerts and determine whether they should've been created. If any were wrongfully created,

I'm not sure if I would phrase the logic this way. If their query was too permissive, the alerts generated wouldn't be "wrongfully created", they'd still be correctly generated based on the query. Perhaps something more like "ensure the rule is generating the alerts you would expect to see".

Reply:
@dplumlee I see what you're saying. Labeling alerts generated from a broadly-scoped query as "wrongfully created", simply they were because they were created in excess, isn't technically correct. How about something like the following: 

```suggestion
If you receive this warning, go to the rule's **Alerts** tab and examine the alerts for anything unexpected. Unexpected alerts might be created from data source issues or queries that are too broadly-scoped.
```